functions/_fzf_search_processes.fish

```diff
@@ -0,0 +1,21 @@
+function _fzf_search_processes --description "Search pid for all running commands"
```

Comment:
Search all running processes. Replace the current token with the id of the selected process."